lib/tile_grid/grid.go

```diff
@@ -47,15 +48,30 @@ func init() {
 }
 
 func setupWorldGrid(data string) error {
+	worldGrid = make(map[string]GlobeIndexSpecialTile)
 	def := GlobalDef{}
 	err := json.Unmarshal([]byte(data), &def)
 	if nil != err {
 		return err
 	}
-	worldGrid = make(map[string]GlobeIndexSpecialTile)
 	for i, tile := range def.GlobeIndexSpecialTiles {
 		worldGrid["tile"+strconv.Itoa(i)] = tile
 	}
+
```

Comment:
does this need to stay here?

Reply:
i just moved the worldGrid init up a few lines so I can easily comment/uncomment the 2 blocks